src/app/Main.js

```diff
@@ -10,7 +10,7 @@ import ConsoleExports from './utils/ConsoleExports';
 import {serverApiRecordEvent} from 'app/utils/ServerApiClient';
 import * as steem from 'steem';
 import {determineViewMode} from "app/utils/Links";
-
+import config from 'config';
```

Comment:
import { NODE_ENV } from `config`, no? or is `config` an `export default` object?

Reply:
no, you are correct.